docs/en-US/component/input-number.md

```diff
@@ -111,6 +125,7 @@ input-number/with-prefix-suffix
 | readonly ^(2.2.16)          | same as `readonly` in native input               | ^[boolean]                                    | false     |
 | disabled                    | whether the component is disabled                | ^[boolean]                                    | false     |
 | controls                    | whether to enable the control buttons            | ^[boolean]                                    | true      |
+| hover-controls ^(2.9.7)     | display control buttons when mouse moves in      | ^[boolean]                                    | false     |
```

Comment:
It is best to explain that it will only take effect when `controls` is `true` and there is no custom slot.